service/src/integrationTest/resources/application.properties

```diff
@@ -7,6 +7,14 @@ core_case_data.api.url=http://fake-ccd-data-store-api
 
 idam.api.url=http://fake-idam-api
 
-fpl.local_authority_name.mapping=example.gov.uk=>EX
+ccd.ui.base.url=http://fake-url
+
+fpl.local_authority_email_to_code.mapping=example.gov.uk=>EX
```

Comment:
We should make LA code here match other mappings. Change to 'example' here or change 'example' to 'EX' in remaining mappings.